frontend/src/components/AnnotationView/AllianceGenomeCard.vue

```diff
@@ -0,0 +1,273 @@
+
+      const matchResult = this.model.name.match(regex);
+
+      if (matchResult) {
+        return matchResult[0];
+      }
+
+      return '';
+    },
+    associatedHumanDiseasesStyle() {
+      if (Object.keys(this.associatedHumanDiseasesData).length === 0) {
+        return {color: `var(--rosalution-grey-300)`};
+      }
+
+      return 'color: black';
+    },
+    associatedPhenotypesStyle() {
+      if (Object.keys(this.associatedPhenotypesData).length === 0) {
+        return {color: `var(--rosalution-grey-300)`};
+      }
+
+      return 'color: black';
+    },
+    experimentalConditionStyle() {
+      if (Object.keys(this.experimentalConditions).length === 0) {
+        return {color: `var(--rosalution-grey-300)`};
+      }
+
+      return 'color: black';
+    },
+  },
+  methods: {
+    calculateAssociatedHumanDiseases() {
+      this.associatedHumanDiseasesData = this.model.diseaseModels;
+    },
+    calculateAssociatedPhenotypes() {
+      const phenotypesDict = {'':
+                {
+                  phenotypes: [],
+                  style: '',
+                  icon: '',
+                  iconStyle: {},
+                },
+      };
+
+      this.model.phenotypes.forEach((phenotype) => {
+        phenotypesDict[''].phenotypes.push(phenotype);
+      });
+
+      this.frequentTermsObject.forEach((term) => {
+        phenotypesDict[term.term] = {
+          phenotypes: [],
+          style: term.style,
+          icon: term.icon,
+          iconStyle: term.iconStyle,
+        };
+      });
+
+      this.model.phenotypes.forEach((phenotype) => {
+        this.frequentTermsObject.forEach((term) => {
+          const regex = new RegExp('\\b' + term.term + '\\b', 'i');
+
+          if (phenotype.match(regex)) {
+            phenotypesDict[term.term].phenotypes.push(phenotype);
+            phenotypesDict[''].phenotypes.splice(phenotypesDict[''].phenotypes.indexOf(phenotype), 1);
+          }
+        });
+      });
+
+      Object.keys(phenotypesDict).forEach((key) => {
+        if (!phenotypesDict[key].phenotypes.length) {
+          delete phenotypesDict[key];
+        }
+      });
+
+      this.associatedPhenotypesData = phenotypesDict;
+    },
+    calculateExperimentalCondition() {
+      if (Object.keys(this.model.conditions).length == 0) {
+        return;
+      }
+
+      this.experimentalConditions = this.model.conditions.has_condition;
+    },
+  },
+});
+
+</script>
+
+<style scoped>
+
+div {
+  font-family: "Proxima Nova", sans-serif;
+}
+
+ul {
+  list-style-type: disc;
+  padding-left: 7%;
+}
+
+li {
+  margin-left: 10px;
+  padding-top: 1%;
+  font-size: 15px;
+}
+
+.card-header {
+    font-size: 22px;
+    font-weight: bold;
+}
+
+.card-sub-header {
+    font-size: 16px;
+    padding-top: 2%;
+}
+
+.card-base {
+  position: relative;
+  text-decoration: none;
+  width: 30rem;
+  padding: var(--p-8);
+  background-color: var(--rosalution-grey-50);
+  display: block;
```

Comment:
This `display: block` can get removed since the `.card-base` class is being applied to either a `div` or a `span` dom elements, they both default to `display: block` in user agent style sheets in the browsers we support, so there isn't a need for the repetition of including it again.  this can help consolidate the CSS to make it easier to modify & traverse.

Reply:
<img width="649" alt="Screenshot 2023-06-05 at 10 58 00 AM" src="https://github.com/uab-cgds-worthey/rosalution/assets/1209059/a1830d6b-1a69-48d1-a67d-07d7030e1cae">

Reply:
Two instances of `display: block` have been removed from `.card-base`